<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>651</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="button1">
         <property name="text">
          <string>글씨체 바꾸기</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button2">
         <property name="text">
          <string>파란색으로 바꾸기</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button3">
         <property name="text">
          <string>내용 지우기</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button4">
         <property name="text">
          <string>크기 키우기</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button5">
         <property name="text">
          <string>크기 줄이기</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
